userspace/libsinsp/plugin.cpp

```diff
@@ -1001,6 +1002,12 @@ bool sinsp_plugin::extract_fields(sinsp_evt* evt,
 	in.owner = (ss_plugin_owner_t*)this;
 	in.get_owner_last_error = sinsp_plugin::get_owner_last_error;
 	in.table_reader_ext = &table_reader_ext;
+	in.field_offsets = field_offsets;
+	if(in.field_offsets) {
+		in.request_offsets = true;
+	} else {
+		in.request_offsets = false;
+	}
```

Comment:
This sort of makes the `request_offsets` field redundant, doesn't it? It just means `field_offsets != nullptr`